src/common/components/hosts/hardwareInfo.ts

```diff
@@ -26,7 +26,7 @@ export const getDiskCapacity = (inventory: Inventory): number =>
     .reduce((diskSize: number, disk: Disk) => diskSize + (disk.sizeBytes || 0), 0) || 0;
 
 export const getHumanizedCpuClockSpeed = (inventory: Inventory) =>
-  Humanize.formatNumber(inventory.cpu?.frequency || 0);
+  Humanize.formatNumber(inventory.cpu?.clockMegahertz || 0);
```

Comment:
```suggestion
  Humanize.formatNumber(inventory.cpu?.clockMegahertz || inventory.cpu?.frequency || 0);
```

Reply:
On Thursday I already had these changes made but I couldn't upload them because Github is not working well